src/lib/sheet/log.ts

```diff
@@ -0,0 +1,26 @@
+import { Post } from '../../types/post'
+import { Recipient } from '../../types/recipient'
+import { getRecipientGroupById } from '../firestore/recipientGroup'
+import { insertRow } from './insert'
+
+export const insertLog = async (user: string, action: string, target: string) => {
+  // const resource = {
+  //   values: [[new Date().toLocaleString(), user, action, target]],
+  // }
+  // db.spreadsheets.values.append({
+  //   spreadsheetId: sheetId,
+  //   range: 'シート1!A1',
+  //   requestBody: resource,
+  // })
+  //to add first row
+  insertRow(1, [new Date().toLocaleString(), user, action, target])
+}
+
+export const recipientSummary = async (recipient: Recipient) => {
+  const groupName = (await getRecipientGroupById(recipient.recipientGroupId)).name
+  return `受給団体${groupName}, 受給者${recipient.name}`
```

Comment:
冗長なので、実データを繋げるだけで良いかと。
```suggestion
  return `${groupName}_${recipient.name}`
```

Reply:
投稿の概要も繋げただけにしました。基本的に余計なデータは付与せず繋げて表示するかたちで良いでしょうか？

Reply:
はい。そうですね。1度内容が分かれば、冗長だと思うので。